frontend/public/components/utils/request-size-input.tsx

```diff
@@ -2,32 +2,37 @@ import * as React from 'react';
 import { Dropdown } from './dropdown';
 import * as classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
+import { NumberSpinner } from './number-spinner';
 
 export const RequestSizeInput: React.FC<RequestSizeInputProps> = ({
   children,
   defaultRequestSizeUnit,
   defaultRequestSizeValue,
   describedBy,
   dropdownUnits,
-  inputClassName,
+  // inputClassName,
```

Comment:
We should go ahead and remove this instead of just commenting it out if it's not needed anymore. Do you know if any code is using this prop?

Reply:
Good catch. I'll don't think this prop is being used, but since it has been a little bit, I'll double check.

Reply:
Yeah, I suspect TypeScript would catch it if it was. It should be safe to remove if it builds when commented out.